login_page/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>login_page</title>
+    <link rel="stylesheet" href="/reset.css"/>
+    <link rel="stylesheet" href="/login_page/style.css" />
+    <link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/variable/pretendardvariable.min.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css" crossorigin="anonymous">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+<body>
+    <div class="container">
+        <a class="panda_logo" a href="/index.html">
+            <picture>
+                <source media="(max-width: 768px)" srcset="/login_page/login_image/mobile/panda_logo_mobile.png">
+                <img src="/login_page/login_image/logo.png" alt="판다마켓 로고">
+            </picture>
+        </a>
+        <div class="input_sector">
+            <div class="email_sector">
+                <span>이메일</span>
+                <div class="inputbox">
+                    <input type="email" placeholder="codeit@email.com" >
+                </div>
+            </div>
+            <div class="pw_sector">
+                <span>비밀번호</span>
+                <div class="inputbox">
+                    <input type="password" placeholder="">
+                    <div class="eye">
+                        <i class="fas fa-eye"></i>
+                    </div>
+            </div>
+        </div>
+        <div class="login_btn">로그인</div>
```

Comment:
❗️ 수정요청
해당 태그는 로그인 동작을 수행하는 태그이므로 button 태그가 더 적절합니다.
즉 form에서 제출 이벤트가 발생했을때 해당 form 안의 유저가 입력한 정보를 제출하는 역할을 하므로
button으로 작성해주세요!

```suggestion
                <button type="submit" class="login_btn">로그인</button>
```

button 태그에서 type 속성은 해당 태그의 동작과 연관이 되어 있어 가능하면 늘 기입하시는 것이 좋습니다.
로그인 페이지의 form에서 로그인 버튼, 회원가입 페이지의 form에서 회원가입 버튼과 같은 경우는 submit 타입이고 그외 대부분의 버튼은 button 타입입니다.